akka-http-core/src/main/scala/akka/http/scaladsl/model/Uri.scala

```diff
@@ -30,6 +30,14 @@ sealed abstract case class Uri(scheme: String, authority: Authority, path: Path,
   def isEmpty: Boolean
 
   /**
+   * Parses the given String into a Query instance.
+   */
+  def query(charset: Charset = UTF8, mode: Uri.ParsingMode = Uri.ParsingMode.Relaxed): Query = queryString match {
```

Comment:
We have
```scala
def apply(input: ParserInput, charset: Charset, mode: Uri.ParsingMode)
def query(charset: Charset = UTF8, mode: Uri.ParsingMode = Uri.ParsingMode.Relaxed)
```

for which we want to provide shortcuts with defaults for `charset` and `mode`. We could